pkg/socketcan/emulator.go

```diff
@@ -32,7 +32,7 @@ type EmulatorOption func(*emulatorCfg)
 
 // WithMulticastAddress sets the address for the multicast group that the Emulator should listen on.
 // A multicast address starts with 239.x.x.x, and using an address that does not conform to this
-// will lead to undefined behaviour.
```

Comment:
I often tend to spell it the English way as I find it more common. Is this not considered the most common way?

Reply:
Linter complaints as well.